en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-example.md

```diff
@@ -0,0 +1,285 @@
+       </call>
+   </sequence>
+```
+
+11. Navigate to **MI Project Explorer** > **APIs** and click on the **+** sign next to APIs to open the **Synapse API Artifact** creation form.
+  
+12. Provide the API name as `SQSAPI` and the API context as `/sqs` and click **Create**. You can go to the source view of the XML configuration file of the API and copy the following configuration. 
+```xml
+   <?xml version="1.0" encoding="UTF-8"?>
+   <api context="/sqs" name="SQSAPI" xmlns="http://ws.apache.org/ns/synapse">
+       <resource methods="POST" uri-template="/sendToQueue">
+           <inSequence>
+               <property expression="json-eval($.queueName)" name="queueName" scope="default" type="STRING" />
+               <sequence key="buildMessage" />
+               <sequence key="createQueue" />
+               <sequence key="sendMessage" />
+               <sequence key="ReceiveAndForwardMessage" />
+               <respond />
+           </inSequence>
+           <faultSequence/>
+       </resource>
+   </api>
+```
+
+Now we can export the artifacts into a Carbon Application (CApp) and deploy it to the server runtime to run it and test it.
+
+## Get the project
+
+You can download the ZIP file and extract the contents to get the project code.
+
+<a href="{{base_path}}/assets/attachments/connectors/amazonSQSExample.zip">
+    <img src="{{base_path}}/assets/img/integrate/connectors/download-zip.png" width="200" alt="Download ZIP">
+</a>
+
+!!! tip
+    You may need to update the value of the access key and make other such changes before deploying and running this project.
+
+## Deployment
+
+Follow the steps in [deploy-artifacts]({{base_path}}/develop/deploy-artifacts) guide to build and export it into a CAR file (`.car` file) using the Micro Integrator for Visual Studio Code extension.
+
+## Test
+
+1. Deploy the back-end service `SimpleStockQuoteService`.
+
+    1. Download the ZIP file of the back-end service from [here](https://github.com/wso2-docs/WSO2_EI/blob/master/Back-End-Service/axis2Server.zip).
+    2. Extract the downloaded zip file.
+    3. Open a terminal and navigate to the `axis2Server/bin/` directory inside the extracted folder.
+    4. Execute the following command to start the axis2server with the `SimpleStockQuote` back-end service:
+
+        === "On MacOS/Linux/CentOS"   
+            ```bash
+             sh axis2server.sh
+            ```
+        === "On Windows"              
+            ```bash
+            axis2server.bat
+            ```
+
+3. Invoke the API as shown below using the curl command. Curl Application can be downloaded from [here](https://curl.haxx.se/download.html).
+
+   ```
+       curl --location 'http://localhost:8290/sqs/sendToQueue' \
+       --header 'Content-Type: application/json' \
+       --data '{
+          "companyName":"WSO2",
+          "queueName":"Queue1"
+       }'
+    ```
+
+**Expected Response**: 
+
+Next, you will receive a response with `sys_id`. Make sure to save it.
```

Comment:
Shall we update this description? It seems irrelevant here

Reply:
Removed this.